release/_bom.sh

```diff
@@ -368,6 +368,72 @@ function generate_pom_release_bom_compile_only {
 	) >> "${pom_file_name}"
 }
 
+function generate_pom_release_bom_test {
+	local pom_file_name="release.${LIFERAY_RELEASE_PRODUCT_NAME}.bom.test-${_ARTIFACT_RC_VERSION}.pom"
+
+	lc_log DEBUG "Generating ${pom_file_name}."
+
+	sed \
+		--expression "s/__ARTIFACT_ID__/release.${LIFERAY_RELEASE_PRODUCT_NAME}.bom.test/" \
+		--expression "s/__ARTIFACT_RC_VERSION__/${_ARTIFACT_RC_VERSION}/" \
+		--expression "s/__GITHUB_REPOSITORY__/liferay-${LIFERAY_RELEASE_PRODUCT_NAME}/" \
+		--expression "s/__PRODUCT_VERSION__/${_PRODUCT_VERSION}/" \
+		--expression "w ${pom_file_name}" \
+		"${_RELEASE_TOOL_DIR}/templates/release.bom.pom.tpl" > /dev/null
+
+	echo "" >> "${pom_file_name}"
+
+	local dependencies_list=(
+		biz.aQute.bnd:biz.aQute.bndlib
+		com.liferay.portal:com.liferay.portal.test
+		com.liferay.portal:com.liferay.portal.test.integration:6.0.36
+		com.liferay:com.liferay.arquillian.extension.junit.bridge
+		com.liferay:com.liferay.arquillian.extension.junit.bridge.connector
+		com.liferay:org.apache.logging.log4j
+		com.liferay:org.apache.logging.log4j.core
+		junit:junit:4.13.1
+		org.apache.aries.jmx:org.apache.aries.jmx.core:1.1.8
+		org.slf4j:log4j-over-slf4j:1.7.25
+	)
+
+	IFS=$'\n' sorted_dependencies_list=($(printf '%s\n' "${dependencies_list[@]}" | sort --field-separator=':' --key=2,2))
+
+	local artifact_urls=$( \
+		find "${_PROJECTS_DIR}/liferay-portal-ee/modules/.releng" -name '*.properties' -print0 | \
+		xargs -0 awk -F= '/^artifact.url=/ { print $2 }')
```

Comment:
@kiwm, how costly is this `find`? Do we really need to run in the directory `liferay-portal-ee/modules/.releng`?

Assuming we already know where the artifacts information is, can we look for the necessary info directly?